tf/src/tf_echo.cpp

```diff
@@ -91,14 +91,25 @@ int main(int argc, char ** argv)
   }
   ros::Rate rate(rate_hz);
 
+  int precision(3);
+  {
```

Comment:
The braces here are not necessary.

Reply:
The point was RAII: after the braces closes `p_nh` is automatically destroyed.